src/enzo/macros_and_parameters.h

```diff
@@ -334,6 +352,7 @@ typedef long long int   HDF5_hid_t;
 #define HDF5_FILE_PINT HDF5_FILE_I8
 #define PISYM "lld"
 #define ENPY_PINT NPY_LONG
+#define EYT_PINT YT_LONG
```

Comment:
Same as my comment about ENPY_PINT and EYT_PINT

Reply:
`ifdef` is added here.